routes/login.js

```diff
@@ -10,9 +10,9 @@ router.post('/login', urlencodedParser, function(req,res){
     console.log(req.body);
     var newUser = new user.User(req.body);
     newUser.save().then(item => {
-        res.redirect('/signin.html');
+        res.send('signup is Successful')
```

Comment:
signup is "s"uccessful
rewrite the message for line 33 to be similar so that messages are consistent